lib/services/censored_log.dart

```diff
@@ -17,17 +17,23 @@ extension CensoredMessage on LogRecord {
       return loginCensoredMessage;
     }
 
-    final finampUserHelper = GetIt.instance<FinampUserHelper>();
-
     String workingLogString = logString;
 
-    for (final user in finampUserHelper.finampUsers) {
-      workingLogString = workingLogString.replaceAll(
-          CaseInsensitivePattern(user.baseUrl), "BASEURL");
-      workingLogString = workingLogString.replaceAll(
-          CaseInsensitivePattern(user.accessToken), "TOKEN");
+    // If userHelper is not initialized, calling code cannot have used baseurl/token
+    // so skipping censoring is fine.
```

Comment:
The problem is that I'm writing the log text to a file as text, so I need to be able to call this function whenever anything is logged, and there's a lot of events that might need logging before userhelper is set up, like hive initialization.  We shouldn't be able to make any connections before userhelper is instantiated because it's the only class that knows the the server url.  We could have leaks while logging in though.

Reply:
Could you try benchmarking how much of an effect this has? Unless it really improves startup times, I'd prefer to at least use a different condition.  
In the login flow I make requests that don't use the UserHelper, but since I do use JellyfinApiHelper, the UserHelper is actually instantiated already. But that's more of a coincidence, and testing a server connection will often leak private info.

Reply:
It's not about performance, its that getting the userhelper throws an error early in startup when it isn't initialized, so I'm forced to skip the actual censoring.  The comment is explaining why this shouldn't result in leaked info under normal circumstances.

The censoring does not work during logging because it's just blanking out the current user's baseURL and token, and we don't have a current user yet.  I'm unsure of a good strategy that could handle these sort of cases.

Reply:
Ahh, now I get it. In that case, that approach seems valid. Maybe we can extend the logging framework at some point to indicate sensitive info when actually producing the log, or simply not log such info if it can't be automatically censored. But that's beyond the scope of this PR :)